nbextensions/usability/toc/main.js

```diff
@@ -6,9 +6,9 @@ define(["require", "jquery", "base/js/namespace"], function (require, $, IPython
   var make_link = function (h) {
     var a = $("<a/>");
     a.attr("href", '#' + h.attr('id'));
-    // get the text *excluding* the link text, whatever it may be
     var hclone = h.clone();
-    hclone.children().remove();
+    // last() is the |P anchor which lights up when you hoover over the headline
+    hclone.children().last().remove();
     a.text(hclone.text());
```

Comment:
Ok, text is much more readable...

![2015-11-05_231343](https://cloud.githubusercontent.com/assets/890156/10983472/e561b256-8412-11e5-8ccf-a74546062458.png)